cypress/component/unit_test_apollo_state_updates.spec.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { mount } from '@cypress/react';
+import Dashboard from '../../src/components/organisms/Dashboard';
+import AddComponentButton from '../../src/components/atoms/AddComponentButton';
+import { dashboardItemsVar } from '../../src/cache/index';
+
+describe('Apollo state management', () => {
+    beforeEach(() => {
+        dashboardItemsVar([]);
+    });
+    it('dashboard starts out empty', () => {
+        mount(<Dashboard />);
+        expect(dashboardItemsVar()).to.deep.equal([]);
+    });
+    it('adds new item to state', () => {
+        // This has to change when we add more functionality to the "Add button" button
```

Comment:
Add button button?

Reply:
Sorry bud, I will run grammarly next time. Good spot👯‍♀️

Reply:
Talk to @thomran 🤓